src/sentry/conf/server.py

```diff
@@ -1874,6 +1874,9 @@ def get_sentry_sdk_config():
     KAFKA_INGEST_TRANSACTIONS: {"cluster": "default", "topic": KAFKA_INGEST_TRANSACTIONS},
 }
 
+# If True, consumers will create the topics if they don't exist
+KAFKA_CONSUMER_AUTO_CREATE_TOPICS = True
```

Comment:
I think we should set this to `False` in getsentry?

Reply:
Totally https://github.com/getsentry/getsentry/pull/4480